lib/node_runner.go

```diff
@@ -27,13 +26,12 @@ type NodeRunner struct {
 	connectionManager *sebaknetwork.ConnectionManager
 	storage           *sebakstorage.LevelDBBackend
 
-	handleMessageFromClientCheckerFuncs []sebakcommon.CheckerFunc
-	handleBallotCheckerFuncs            []sebakcommon.CheckerFunc
+	handleTransactionCheckerFuncs []sebakcommon.CheckerFunc
```

Comment:
is the renaming related to the issue?

Reply:
@kfangw A little bit, or deeply. The `MessageFromClient` was removed from `sebaknetwork.MessageType`, and 'message from client' was also too, because the message like transaction can be received from node. So 'message from client' expression was replaced by 'transaction'.